lightning-invoice/src/utils.rs

```diff
@@ -38,29 +39,28 @@ use sync::Mutex;
 ///   may be too long for QR code scanning. To fix this, `PhantomRouteHints::channels` may be pared
 ///   down
 ///
-/// `payment_hash` and `payment_secret` can come from [`ChannelManager::create_inbound_payment`] or
+/// `payment_hash` can come from [`ChannelManager::create_inbound_payment`] or
```

Comment:
Oops, no, we don't want payment hash to come from there anymore - maybe "payment_hash can be specified if you have a specific need for a custom payment hash, see the difference between create_inbound_payment and create_inbound_payment_for_hash" (with links).